src/NewsService.Business/FindNewsCommand.cs

```diff
@@ -16,37 +32,138 @@ public class FindNewsCommand : IFindNewsCommand
     private readonly INewsRepository _repository;
     private readonly INewsInfoMapper _mapper;
     private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IRequestClient<IGetUsersDataRequest> _rcGetUsers;
+    private readonly IRequestClient<IGetDepartmentsRequest> _rcGetDepartments;
+    private readonly ILogger<GetNewsCommand> _logger;
+    private readonly IBaseFindRequestValidator _baseFindValidator;
+
+    private async Task<List<UserData>> GetAuthor(List<Guid> authorIds, List<string> errors)
+    {
+      if (authorIds == null || authorIds.Count == 0)
+      {
+        return null;
+      }
+
+      string errorMessage = "Cannot get authors now. Please try again later.";
+      const string logMessage = "Cannot get authors with ids: {authorIds}.";
```

Comment:
what for const?

Reply:
const don't do this worse. This string will be always in one instance without creating in all time when method is called, but dotnet can do it by himself. But still it doesn't bad

Reply:
but if you added const, so add in prev line too

Reply:
done

Reply:
not done